internal/controllers/metrics.go

```diff
@@ -45,8 +46,10 @@ func (c ApiController) SaveMetricsHandler(w http.ResponseWriter, r *http.Request
 
 	if err != nil {
 		// error on saving in database
-		log.Println(err.Error())
+		log.Errorf("Unable to save metrics for node %v to database, error: %v", requestContext.NodeId, err)
 		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 		return
 	}
+
+	log.Debugf("Node %s saved new metrics", requestContext.NodeId)
```

Comment:
Maybe add saved metrics

Reply:
just important ones like finalized and head block slot